.github/workflows/pull-request.yml

```
@@ -206,9 +206,17 @@ jobs:
         id: changed-files
         uses: tj-actions/changed-files@v45
 
-      - name: Run tests
+      - name: Run Jest Tests
         if: steps.changed-files.outputs.only_changed != 'true'
-        run: npm run test -- --watchAll=false --coverage
+        run: |
+          npm run test
+        exit 1
```

actionlint (1.7.3)

could not parse as YAML: yaml: line 213: could not find expected ':'

(syntax-check)